content-optimization/tests/pipelines/data_processing/test_nodes.py

```
@@ -98,22 +139,24 @@ def test_extract_data(catalog: DataCatalog, word_count_cutoff: int):
             "extracted_headers",
             "extracted_img_alt_text",
             "extracted_content_body",
-        }.issubset(df.columns), "Exptected columns missing in the extracted dataframe"
-
-        # Check if number of articles with extracted content body matches the number of text files
```

Good catch on this change! Just one minor thing, this test case will either always pass or always fail depending on the nature of your mock data. As long as your mock data does not contain an empty string for the `extracted_content_body`, this assertion will always pass. I am assuming yours always failed because of how you created your mock data. Nonetheless, this is a redundant assertion. Good catch!